UnitedIncome/components
# [APP-3020] [APP-3181] SVG Scoping and Optimization

### Description
> Please provide a brief description of your changes below.

This pull request fixes a number of issues we've been having with all of our new SVG illustrations, and as a result fixes a massive bug we've been having in the marketing site where our illustrations were not rendering correctly in IE and Safari.

* Correctly scopes all of the svgs using an id system which makes sense. Instead of using the defaulted export name, instead it now follows `#component-name-1` anytime a clip or fill rule is required. This prevents name clashing and style bleeding between svgs when multiple are used on the same page.
* Re-imported all of the svgs from the first round so their viewbox is now `0 0 140 140`, this should close `APP-3020` which has been open for some time.
* Ran all of the svgs through imageoptim, which has reduced the payload for each svg anywhere between 5-20%.
* Removed inline `<style>` tags in favor for the appropriate svg prop, ie `fill="#fff"`.
* Fixed a typo in the compass story.
* Fixed an issue where the health meter illuminated story wasn't rendering correctly. 

<img width="902" alt="Screen Shot 2019-07-29 at 10 38 57 PM" src="https://user-images.githubusercontent.com/10888441/62098578-efb51900-b258-11e9-91df-17f07cdaa217.png">

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

* Look at each illustration in storybook, they should all look the same as they did before. They should also be the same size which was a bug before (Good way to test this is by looking at the car illustration, and then looking at the lamp, previously they were different sizes).
* Test each illustration in Safari and Firefox also, as these browsers tend to have different opinions on how svgs should render.

### Comments
> Any additional comments that will aid with the review process.

- [JIRA Ticket](https://unitedincome.atlassian.net/browse/app-3020)
- [JIRA Ticket](https://unitedincome.atlassian.net/browse/app-3181)
